tensorflow/core/ops/sparse_ops.cc

```diff
@@ -39,6 +39,51 @@ Status SparseSparseMinOrMaxShapeFn(InferenceContext* c) {
   return Status::OK();
 }
 
+Status SparseReduceShapeFn(InferenceContext* c) {
```

Comment:
This is a super nit, but would you consider putting it here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/common_shape_fns.cc

Reply:
@akshaym Thanks for the suggestion. The shape function has been moved to `common_shape_fns.cc`.